website/docs/tutorials/deploying-java-app-with-database.md

```diff
@@ -0,0 +1,260 @@
+    ```shell
+    odo push --show-logs
+    ```
+
+5. The application is now deployed to the cluster - you can view the status of the cluster, and the application test results by streaming the cluster logs into the terminal.
+    ```shell
+    odo log
+    ```
+
+    Notice the failing tests due to an `UnknownDatabaseHostException`:
+    ```shell
+    [INFO] [err] java.net.UnknownHostException: ${DATABASE_CLUSTERIP}
+    [INFO] [err]    at java.base/java.net.AbstractPlainSocketImpl.connect(AbstractPlainSocketImpl.java:220)
+    [INFO] [err]    at java.base/java.net.SocksSocketImpl.connect(SocksSocketImpl.java:403)
+    [INFO] [err]    at java.base/java.net.Socket.connect(Socket.java:609)
+    [INFO] [err]    at org.postgresql.core.PGStream.<init>(PGStream.java:68)
+    [INFO] [err]    at org.postgresql.core.v3.ConnectionFactoryImpl.openConnectionImpl(ConnectionFactoryImpl.java:144)
+    [INFO] [err]    ... 86 more
+    [ERROR] Tests run: 2, Failures: 1, Errors: 1, Skipped: 0, Time elapsed: 0.706 s <<< FAILURE! - in org.example.app.it.DatabaseIT
+    [ERROR] testGetAllPeople  Time elapsed: 0.33 s  <<< FAILURE!
+    org.opentest4j.AssertionFailedError: Expected at least 2 people to be registered, but there were only: [] ==> expected: <true> but was: <false>
+    at org.example.app.it.DatabaseIT.testGetAllPeople(DatabaseIT.java:57)
+    
+    [ERROR] testGetPerson  Time elapsed: 0.047 s  <<< ERROR!
+    java.lang.NullPointerException
+    at org.example.app.it.DatabaseIT.testGetPerson(DatabaseIT.java:41)
+    
+    [INFO]
+    [INFO] Results:
+    [INFO]
+    [ERROR] Failures:
+    [ERROR]   DatabaseIT.testGetAllPeople:57 Expected at least 2 people to be registered, but there were only: [] ==> expected: <true> but was: <false>
+    [ERROR] Errors:
+    [ERROR]   DatabaseIT.testGetPerson:41 NullPointer
+    [INFO]
+    [ERROR] Tests run: 2, Failures: 1, Errors: 1, Skipped: 0
+    [INFO]
+    [ERROR] Integration tests failed: There are test failures.
+    ```
+   Note: This error will be fixed at a later stage in the tutorial when we connect a database instance to this application.
+
+6. You can also create an ingress URL with `odo` to access the application:
+    ```shell
+    odo url create --host $(minikube ip).nip.io
+    ```
+
+7. Push the URL to activate it:
+    ```shell
+    odo push --show-logs
+    ```
+
+8. Display the created URL:
+    ```shell
+    odo url list
+    ```
+
+    You will see a fully formed URL that can be used in a web browser:
+    ```shell
+    $ odo url list
+    Found the following URLs for component mysboproj
+    NAME               STATE      URL                                           PORT     SECURE     KIND
+    mysboproj-9080     Pushed     http://mysboproj-9080.192.168.49.2.nip.io     9080     false      ingress
+    ```
+
+10. Use the URL to navigate to the `CreatePerson.xhtml` data entry page and enter requested data:
+URL/CreatePerson.xhtml and enter a user's name and age data using the form.
```

Comment:
```suggestion
in case of this tutorial, we will access `http://mysboproj-9080.192.168.49.2.nip.io/CreatePerson.xhtml`. Note that the URL could be different for you. Now, enter a name and age data using the form.
```